include/mme-app/mmeSmDefs.h

```diff
@@ -36,86 +36,102 @@ const uint16_t attach_wf_init_ctxt_resp_att_cmp = 12;
-const uint16_t MME_INIT_DETACH = 131; 
-const uint16_t PURGE_RESP_FROM_HSS = 132; 
-const uint16_t REL_AB_RESP_FROM_SGW = 133; 
-const uint16_t S1_REL_REQ_FROM_UE = 134; 
-const uint16_t SEC_MODE_RESP_FROM_UE = 135; 
-const uint16_t SERVICE_REQUEST_FROM_UE = 136; 
-const uint16_t TAU_NOT_REQUIRED = 137; 
-const uint16_t TAU_REQUEST_FROM_UE = 138; 
-const uint16_t TAU_REQUIRED = 139; 
-const uint16_t UE_CTXT_REL_COMP_FROM_ENB = 140; 
-const uint16_t ULA_FROM_HSS = 141; 
-const uint16_t VALIDATE_IMSI = 142; 
-const uint16_t END_EVENT = 143;    
+const uint16_t ACT_DED_BEARER_ACCEPT_FROM_UE = 101; 
+const uint16_t ACT_DED_BEARER_REJECT_FROM_UE = 102; 
+const uint16_t AIA_FROM_HSS = 103; 
+const uint16_t ATT_CMP_FROM_UE = 104; 
+const uint16_t ATTACH_REQ_FROM_UE = 105; 
+const uint16_t AUTH_RESP_FAILURE = 106; 
+const uint16_t AUTH_RESP_FROM_UE = 107; 
+const uint16_t AUTH_RESP_SUCCESS = 108; 
+const uint16_t AUTH_RESP_SYNC_FAILURE = 109; 
+const uint16_t CLR_FROM_HSS = 110; 
+const uint16_t CREATE_BEARER_REQ_FROM_GW = 111; 
+const uint16_t CREATE_BEARER_START = 112; 
+const uint16_t CS_RESP_FROM_SGW = 113; 
+const uint16_t DDN_FROM_SGW = 114; 
+const uint16_t DED_ACT_COMPLETE = 115; 
+const uint16_t DEL_SESSION_RESP_FROM_SGW = 116; 
+const uint16_t DETACH_ACCEPT_FROM_UE = 117; 
+const uint16_t DETACH_REQ_FROM_UE = 118; 
+const uint16_t ENB_STATUS_TRANFER_FROM_SRC_ENB = 119; 
+const uint16_t eRAB_MOD_IND_START = 120; 
+const uint16_t ERAB_MOD_INDICATION_FROM_ENB = 121; 
+const uint16_t ERAB_SETUP_RESP_FROM_ENB = 122; 
+const uint16_t ESM_INFO_NOT_REQUIRED = 123; 
+const uint16_t ESM_INFO_REQUIRED = 124; 
+const uint16_t ESM_INFO_RESP_FROM_UE = 125; 
+const uint16_t GW_CP_REQ_INIT_PAGING = 126; 
+const uint16_t GW_INIT_DED_BEARER_AND_SESSION_SETUP = 127; 
+const uint16_t HO_CANCEL_REQ_FROM_SRC_ENB = 128; 
+const uint16_t HO_FAILURE_FROM_TARGET_ENB = 129; 
+const uint16_t HO_NOTIFY_FROM_ENB = 130; 
+const uint16_t HO_REQUEST_ACK_FROM_ENB = 131; 
+const uint16_t HO_REQUIRED_FROM_ENB = 132; 
+const uint16_t IDENTITY_RESPONSE_FROM_UE = 133; 
+const uint16_t IMSI_VALIDATION_FAILURE = 134; 
+const uint16_t IMSI_VALIDATION_SUCCESS = 135; 
+const uint16_t INIT_CTXT_RESP_FROM_UE = 136; 
+const uint16_t INTRA_S1HO_START = 137; 
+const uint16_t MB_RESP_FROM_SGW = 138; 
+const uint16_t MME_INIT_DETACH = 139; 
+const uint16_t PAGING_COMPLETE = 140; 
+const uint16_t PURGE_RESP_FROM_HSS = 141; 
+const uint16_t REL_AB_RESP_FROM_SGW = 142; 
+const uint16_t S1_REL_REQ_FROM_UE = 143; 
+const uint16_t SEC_MODE_RESP_FROM_UE = 144; 
+const uint16_t SERVICE_REQUEST_FROM_UE = 145; 
+const uint16_t TAU_NOT_REQUIRED = 146; 
+const uint16_t TAU_REQUEST_FROM_UE = 147; 
+const uint16_t TAU_REQUIRED = 148; 
+const uint16_t UE_CTXT_REL_COMP_FROM_ENB = 149; 
+const uint16_t ULA_FROM_HSS = 150; 
+const uint16_t VALIDATE_IMSI = 151; 
+const uint16_t END_EVENT = 152;    
 
 /******************************************
 * Maps Event Name to Event ID
```

Comment:
wondering why no change for "event name" to event Id ??

Reply:
New Events w.r.t the Create Bearer Procedure and Dedicated Bearer Activation are in place(Sorted alphabetically as per the scripts).